addons/account_peppol/wizard/peppol_registration.py

```diff
@@ -20,6 +19,37 @@ class PeppolRegistration(models.TransientModel):
         required=True,
         default=lambda self: self.env.company,
     )
+    is_branch_company = fields.Boolean(
```

Comment:
`is_branch_company`, `active_parent_company` and `can_use_parent_connection` are stored uselessly imo. Just turn them into computed non-stored fields with a depends. Or avoid having them if possible.